packages/product-components/src/components/PassphraseTypeCard/PassphraseTypeCardContent.tsx

```diff
@@ -43,12 +44,18 @@ const Description = styled.div`
 
 type PassphraseTypeCardContentProps = {
     submitLabel: ReactNode;
+    submitVariant?: 'primary' | 'warning';
```

Comment:
@adamhavel is it better to allow strictly just those two variants here or should we rather use:
```suggestion
    submitVariant?: ButtonVariant;
```

same for banner `BannerVariant`

Reply:
Definitely there is now an inconsistency between `PassphraseTypeCardContent` and `NonAsciiBanner`, I'll use `ButtonVariant` at both places.

Reply:
https://github.com/trezor/trezor-suite/pull/15238/commits/02ebd9a7bcb4cf72732beba84150775e945f5cc8